server/config/config.go

```diff
@@ -124,6 +124,7 @@ type ServerConfig struct {
 	CompactionBatchLimit    int
 	CompactionSleepInterval time.Duration
 	QuotaBackendBytes       int64
+	InitialMmapSize       	uint64
```

Comment:
Nit - Alignment of `unit64` here looks slight off.

Reply:
Done. :) thanks for your review.